### ISSUE TITLE
Enable Dialogue concerning stock amounts

### ISSUE THREAD

**Issue by komizutama** [openpantry/open_pantry#116]
Test several design patterns for the following dialogue
- [ ] If the guest has insufficient credits, a message is posted that says "We're sorry, you have insufficient noms please select a lower amount. Thank you!"  
- [ ] Notify guest if trying to add an item that is not available.

This can be a modal, error message in the window or whatever makes sense.

**Comment by fcapovilla** [comment]
@komizutama Right now, when the user has insufficient credits to add an item, the plus button is disabled and replaced by a "Max allowed" text, and it is not possible to add the item, which means the error dialog would never be shown, unless we let the user click on the "Max allowed" text, which will only display the error dialog.
The same thing happens for unavailable items.

I understand it would be a good idea to clearly show the user the reason why they cannot add more of an item, but I'm not sure showing a dialog on click is the best solution.

**Comment by bglusman** [comment]
Yeah definitely agree not Dialog/modal but also that ticket probably should have been cleaned up/de-priototized behind thia's other UI work since ember changes there inform best decision ... Apolou, I didn't see which issues you'd self assigned
